File: FASE 1 - Variáveis e Tipos de Dados/Desafio 1/cadastroSimples.cs

//GPTando - Mini-Missões - Fase 1 - Missão 1

using System;

class CadastroSimples {
    static void Main (){
        Console.Write ("Digite seu nome: ");
        string nome = Console.ReadLine();
        
        Console.Write ("Digite sua idade: ");
        int idade = int.Parse(Console.ReadLine()); //colocar o int.Parse() faz converter o resultado digitado em inteiro e voltar como uma string, já que o Console.ReadLine volta sempre uma string, então sem ele, daria erro se o usuário escrevesse um inteiro. Assim, dá erro se ele não escrever um inteiro.
        
        Console.Write ("Olá, " + nome + "! Você tem " + idade + " anos.");
    }
}

File: FASE 1 - Variáveis e Tipos de Dados/Desafio 1/cadastroSimples.java

//GPTando - Mini-Missões - Fase 1 - Missão 1

import java.util.Scanner;

public class CadastroSimples {
    public static void main (String[] args){
       Scanner entrada = new Scanner (System.in);
       
       System.out.print ("Digite seu nome: ");
       String nome = entrada.nextLine();
       
       System.out.print ("Digite sua idade: ");
       String idade = entrada.nextLine();
       
       System.out.println ("Olá, " + nome + "! Você tem " + idade + " anos.");
    }
}
